pkg/controller/template/test_data/templates/aws/master/units/kubelet.service

```diff
@@ -12,6 +12,9 @@ contents: |
         --bootstrap-kubeconfig=/etc/kubernetes/kubeconfig \
         --kubeconfig=/var/lib/kubelet/kubeconfig \
         --rotate-certificates \
+        --container-runtime=remote \
+        --container-runtime-endpoint=unix:///var/run/crio/crio.sock \
+        --runtime-request-timeout=10m \
```

Comment:
Why did you need to set this?  Did you hit an error without it, or is it just increasing robustness?  What's the default value?  How did you decide on 10m?

Reply:
default is 2m.  probably doesn't need to be changed.  i think the cri-o docs recommend this, but i don't think it is needed.

Reply:
> default is 2m. probably doesn't need to be changed

I'm in favor of dropping it then.  The fewer local opinions we need to maintain, the better :).

Reply:
Yeah, I took that from this recommendation: https://github.com/kubernetes-sigs/cri-o/blob/master/kubernetes.md#preparing-kubelet

Reply:
Let's keep the time out then, as it might be required when we pull hyberkube image on slower network.